src/librustdoc/clean/types.rs

```diff
@@ -767,7 +766,7 @@ impl Item {
                             s
                         }),
                     }
-                } else if attr.has_any_name(ALLOWED_ATTRIBUTES) {
+                } else if attr.has_any_name(ALLOWED_ATTRIBUTES) || matches!(attr, hir::Attribute::Parsed(AttributeKind::NoMangle(..))) {
```

Comment:
Since .ident() doesn't work on parsed attributes this check is needed.
@jdonszelmann is the broken .ident() at `compiler/rustc_hir/src/hir.rs:1269` deliberate?

Reply:
so this is exactly the place where you could pretty print those 4 allowed attributes, from the other comment

Reply:
and soon the other 3 should become parsed too ofc